### PR TITLE
Bump scikit-learn compatibility

### PR DESCRIPTION
**Pull request by matzhaugen**
Thanks for a great repo. I see that it is not being actively developed anymore. Not sure if this is of interest but I figured I'd offer thsi compatibiity fix. Feel free to modify.

Bump scikit-learn to `scikit-learn==1.1.3`

Please test on local environment. 

I also see that there is no py3.7-3.10 comp. Any plans to add this? Should be fairly straight forward to just modify the .travis file if it still builds on merges.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Cython>=0.24
 nilearn>=0.2.4
 numpy>=1.12.1
-scikit-learn>=0.19
+scikit-learn>=1.1.13
```
Comment (abelowska) [inline requirements.txt:4]: It looks like a typo, I think this should be `1.1.3` instead since there's no published sklearn package with `1.1.13` tag

**Comment by abelowska** [comment]
Thanks for the PR, I've tested it with Python 3.8 and it works fine for me! 

@mnarayan  Are there any plans for maintaining this lib and releasing the updated version?

**Comment by mnarayan** [comment]
Thanks for the PR (@matzhaugen) and the review (@abelowska). 

We would love to continue maintaining this library, but cannot commit to releasing an update on a timeline right now.

**Comment by adamklein** [comment]
I see skggm2 on pypi - is that released from this repo? Looks like skggm 0.2.10 on pypi may just be picking up this bugfix PR if I’m tracking correctly

**Comment by mnarayan** [comment]
@adamklein yes I didn't make that but @matzhaugen did.

**Pull request merged**